IdentityCore/src/oauth2/aad_base/MSIDAADTokenResponse.m

```diff
@@ -90,7 +92,11 @@ - (NSDictionary *)jsonDictionary
     json[MSID_OAUTH2_SUB_ERROR] = self.suberror;
     json[@"adi"] = self.additionalUserId;
     json[MSID_OAUTH2_CLIENT_INFO] = self.clientInfo.rawClientInfo;
-    if (!self.error) json[MSID_OAUTH2_EXT_EXPIRES_IN] = [@(self.extendedExpiresIn) stringValue];
+    if (!self.error)
+    {
+        json[MSID_OAUTH2_EXT_EXPIRES_IN] = [@(self.extendedExpiresIn) stringValue];
+        json[@"ext_expires_on"] = [@(self.extendedExpiresIn) stringValue];
```

Comment:
this should take self.extendedExpiresOn instead of self. self.extendedExpiresIn

Reply:
fixed